src/core/tsi/ssl_transport_security_utils.cc

```diff
@@ -247,4 +248,40 @@ tsi_result SslProtectorUnprotect(const unsigned char* protected_frames_bytes,
   return result;
 }
 
+int verify_crl_signature(X509_CRL* crl, X509* issuer) {
+  EVP_PKEY* ikey = X509_get_pubkey(issuer);
```

Comment:
Please check that `issuer` is not null before passing it into `X509_get_pubkey`.